modules/media/jsx/mediaIndex.js

```diff
@@ -127,8 +129,18 @@ class MediaIndex extends React.Component {
     }
 
     if (column === 'Edit Metadata') {
-      const editURL = loris.BaseURL + "/media/edit/?id=" + row['Edit Metadata'];
-      return <td className={classes}><a href={editURL}>Edit</a></td>;
+      let editButton = (
+        <Modal
+          title="Edit Media File"
+          trigger={<button>Edit</button>}
```

Comment:
i think maybe the js file needs to be recompiled? the modal component here doesn't specify a throwWarning props but the form is still throwing the error modal window